cnf/make.conf.example

```diff
@@ -291,6 +291,20 @@
 #     unset.
 #PORTAGE_IONICE_COMMAND="ionice -c 3 -p \${PID}"
 #
+# PORTAGE_SCHEDULING_POLICY allows changing the current scheduling policy 
+# (defaults to 'other'). The supported options are 'other', 'batch', 'idle',
```

Comment:
Both here and in the man page, could you include a suggestion for what people should use for minimal Portage disruption to the rest of the system?

(What has the effect PORTAGE_NICENESS=19 is supposed to have?)

Reply:
I will add this information, modify a bit tthe variable names and force-push the branch.